services/infrasetservice/helper.go

```diff
@@ -21,7 +21,7 @@ func generateCertsforHarbor() {
 
 	log.Println("CHECK 1")
 	// Delete the directory if it already exists
-	if _, err := os.Stat(path); os.IsExist(err) {
+	if _, err := os.Stat(path); err==nil {
```

Comment:
what if errr!=nil? 
```
usually coding practice is if (err) {
    // Error handing
}

err==nil should not be used unless like iske alawa tareeka hi nai hai code ko aage badhane ka 
```

Reply:
> what if errr!=nil?
> 
> ```
> usually coding practice is if (err) {
>     // Error handing
> }
> 
> err==nil should not be used unless like iske alawa tareeka hi nai hai code ko aage badhane ka 
> ```

yes this error will be handled as mentioned in this post

> https://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go